backend/src/test/java/com/bakdata/conquery/apiv1/frontend/FEValueTest.java

```diff
@@ -24,5 +25,7 @@ public void test() {
 		assertThat(filter.add(secondA)).isFalse();
 
 		assertThat(filter.add(similarA)).isTrue();
+
+		assertThat(Stream.of(firstA, secondA, similarA).distinct()).containsExactly(firstA, similarA);
```

Comment:
An sich testest du hier ob Equals und Hash Code funktionieren und ob HashSet die funktionen benutzt

Reply:
Ja, das war eher Beweisführung dafür, dass die Annotationen so funktionieren wie ich es möchte.

Reply:
Ich hab in den Test mal reingeschrieben warum er da ist/bleiben sollte.